### PR TITLE
6561 added test configuration fo jenkins deplyment in docker

### PR DESCRIPTION
**Pull request by galiaf95**

### CODE REVIEW

File: templates/examples/docker_jenkins/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+
+# Preparin dependencies
+RUN apt-get -y install wget
+RUN apt-get -y install software-properties-common
+RUN add-apt-repository ppa:webupd8team/java -y
+RUN apt-get update
+RUN echo oracle-java7-installer shared/accepted-oracle-license-v1-1 select true | sudo /usr/bin/debconf-set-selections
+RUN apt-get -y install oracle-java7-installer
```
Comment (timofeyturenko) [inline templates/examples/docker_jenkins/Dockerfile:9]: why Oracle java? would openjdk be better?  (no need to add repositories, it is available from standard Ubuntu repos)

File: templates/examples/docker_jenkins/startup.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# cloning mdbci repo
+git clone https://github.com/OSLL/mdbci.git
+
+# Running jenkins
+java -jar jenkins.war --httpPort=5555
```
Comment (timofeyturenko) [inline templates/examples/docker_jenkins/startup.sh:5]: how can all Jenkins plugins be installed here?

File: templates/examples/docker_jenkins/Dockerfile (hunk 1 above)
Comment (galiaf95) [inline templates/examples/docker_jenkins/Dockerfile:9]: When i try to install openjdk it asking for more dependencies like jre, jre-headless and so on.

File: templates/examples/docker_jenkins/startup.sh (hunk 2 above)
Comment (galiaf95) [inline templates/examples/docker_jenkins/startup.sh:5]: It is possible, we need only wget some .hpi file and put them in jenkins plugin directory.

Comment (krinkin) [inline templates/examples/docker_jenkins/startup.sh:5]: What is hpi?

С уважением,
Кирилл Кринкин

Comment (galiaf95) [inline templates/examples/docker_jenkins/startup.sh:5]: Jenkins plugin: https://wiki.jenkins-ci.org/display/JENKINS/Plugins#Plugins-Byhand

File: templates/examples/docker_jenkins/Dockerfile (hunk 1 above)
Comment (krinkin) [inline templates/examples/docker_jenkins/Dockerfile]: Who uses jdk?

**Pull request merged**